dds/DCPS/TopicImpl.cpp

```diff
@@ -146,7 +146,8 @@ TopicImpl::enable()
                                              topic_name_.c_str(),
                                              type_name_.c_str(),
                                              qos_,
-                                             type_support_->has_dcps_key());
+                                             type_support_ ? type_support_->has_dcps_key() : false,
```

Comment:
When can type_support_ be null?  Could this be checked earlier (and we bail out) or is it valid?